Resources/ServerInfo/Guidebook/ReferenceTables/AlertLevels.xml

```diff
@@ -0,0 +1,69 @@
+<Document>
+    # Alert Levels
+    Alert levels can be set at communcations consoles.
```

Comment:
```suggestion
    Alert levels can be set at communications computers.
```

typo. also it's called `communications computer` (see [here](https://github.com/space-wizards/space-station-14/blob/9b40bcc1d9cd63938ccd9833628c23761feef456/Resources/Prototypes/Entities/Structures/Machines/Computers/computers.yml#L560C9-L560C32))